test/unit/view_models/breadcrumb_tests.rb

```diff
@@ -0,0 +1,44 @@
+require "assert"
+require "much-rails/view_models/breadcrumb"
+
+class MuchRails::ViewModels::Breadcrumb
+  class UnitTests < Assert::Context
+    desc "MuchRails::ViewModels::Breadcrumb"
+    subject { unit_class }
+
+    let(:unit_class) { MuchRails::ViewModels::Breadcrumb }
+
+    let(:name) { "name" }
```

Comment:
Move everything here and down into an `InitTests` context please since these tests are about initialized units. Also add the `should have_readers :name, url` and `should have_imeths :render_as_link?` macros to that context.